src/site/content/en/blog/extending-workbox/index.md

```diff
@@ -0,0 +1,379 @@
+---
+title: Extending Workbox
+subhead: >
+  Write your own reusable, shareable strategies and plugins.
+description: >
+  Write your own reusable, shareable strategies and plugins.
+date: 2020-12-10
+thumbnail: thumbnail.png
+hero: hero.png
+alt: The Workbox logo.
+draft: true
+tags:
```

Comment:
Are there any other tags to use for the CDS-related blog posts?

Reply:
There's no CDS-specific posts but I'll check if there are other general tags that are relevant here: https://github.com/GoogleChrome/web.dev/blob/master/src/site/_data/tagsData.json